script.js

```diff
@@ -0,0 +1,47 @@
+const removeBtn = document.createElement('button');
+const addbtn = document.querySelector('.book__add-btn');
+const bookTitle = document.querySelector('.book__title');
+const bookOwner = document.querySelector('.book__owner');
+const book = document.querySelector('.book');
+const bookAuthor = document.getElementsByClassName('book__author');
+const bookName = document.getElementsByClassName('book__name');
+const bookItem = document.querySelector('.book__item');
+
```

Comment:
- [ ] Please make sure that added books are stored in local storage as requested in the project requirements. After refreshing the page users should be able to see the previously added books.
![image](https://user-images.githubusercontent.com/83803414/191226725-fd010e75-cbb2-4bc5-b16c-28e7f766dc73.png)